### PR TITLE
Completed two passing tests on reducers using Jest

### PR DESCRIPTION
**Pull request by chaomonica**

### CODE REVIEW

**Review by Doodlis — DISMISSED** [review]
My bad with leaving the debbuger in there! I understand that fix, but everything else in my files should not have been touched by anyone else, but me. As now there will be more merge conflicts and you revereted some code to the previuos state which is inncorrect.

**Comment by chaomonica** [comment]
I couldn't even make a commit after your merge because the formatting was not passing pomander.  Code should pass a linter before a merge so that other people don't have to fix single quotes and camel case in order to make a commit.  Please let me know which specific areas you would like reverted and which will at the same time still allow me to pass the linter.

**Comment by Doodlis** [comment]
I really do not know what is going on at this moment, and why your files a set up differently than mine, but there are no issues with linter for me and everything works fine, as if it did't work I wouldn't be able to code as well. So not sure where that linter issue is coming from.

**Comment by Doodlis** [comment]
the app works for Jonathan too, so not sure what other changes you have that the rest of us don't :(

**Comment by chaomonica** [comment]
Ahhhh just read your conversation with Rick from last night on slack and I understand what's going on now.  You misunderstand me - I never said I had any trouble with your code not compiling.  The issue I got after you merged your branch was that (as I mentioned in my previous comment), "I couldn't even make a commit after your merge because the formatting was not passing pomander. "  

From what I can see after you trying to fix your linter last night with Rick, a simple fix might be to just install pomander.  
- It would point out the parsing error in auth0-provider-with-history-js on line 12 (which I commented out since you wrote it and you would be able to best fix it whereas I just needed to be able to commit).  
- You would also be able to see all the double quotes in your import statements that I had to manually change to be able to commit.
- Also the camel case on line 11 in Signup.js which in retrospect, I could have just commented out instead of fixing since you might prefer a different naming convention.

I can go ahead and revert the changes that I made and comment them out instead (otherwise pomander prevents me from committing).  Please fix the linter errors when you have time.

**Comment by Doodlis** [comment]
I honestly  thought that you just misnamed linter as pomander once by
accident, because I do not think we made a decision to use it.  I fixed
some of the quotes and all, but somehow I or we all will have to configure
eslint for everyone to be the same for this repo. And as far as pomander
goes, I would just delete it.

**Comment by Doodlis** [comment]
Also, this may be an option: To skip Pomander, commit with the --no-verify
option.

**Comment by chaomonica** [comment]
The --no-verify option sounds like a good option for pomander although I think pomander is a moot point - even with Rick's linter, it was catching the issues he mentioned in yall's slack thread last night.  Like you just mentioned, eslint needs to be the same for everyone so that people are not having to fix other's linting issues.

**Comment by Doodlis** [comment]
I think you are the only one that decided to use pomander, otherwise no one
else needs to fix anyone elses code issues in their files to be able to
commit.

**Comment by chaomonica** [comment]
Like I said previously, pomander is a moot point. People shouldn't be submitting code that doesn't pass the linter that Rick set up.  As I can see in the slack thread, your double quotes wasn't passing the linter that was set up for this project.

**Comment by Doodlis** [comment]
That is the issue! Eslint may not be configured properly  in our repo and
you are using another linter on top of it and it causes you problems, not
anyone else.

**Comment by chaomonica** [comment]
It's not just causing me problems, your double quotes was showing up as having issues for Rick's linter as I can see in your slack thread with him from last night. 
edited: double quotes, not single quotes
image

**Comment by Doodlis** [comment]
Yes, but it doesn't stop him from writing code in his files and commiting
it without messing in filed! There is an Eslint issue that is going on, but
for you we also have a pomander issue. Two issues.

**Comment by Doodlis** [comment]
Messing in my files or anyone elses

**Comment by chaomonica** [comment]
I think if we want to address issues there are a few more issues that need to be addressed.

1. Your linter configuration.
2. You messed with my main page routing which was merged before your was which I really don't see as an issue since this is a group project.
3. Your merging of pull requests without peer review which is supposed to be followed as outlined in the git workflow on Trello.

**Comment by Doodlis** [comment]
Again,  it is not my linter configuration it is the projects linter
configuration. The routes for login and main page can be the same, because
they are rendered conditionally. I said I am sorry and that I rushed the
merge by accident.

**Comment by chaomonica** [comment]
Changes have been pulled from development.  Thanks!

File: front-end/src/store/store.jsx [hunk 1]
```diff
@@ -1,7 +0,0 @@
-import { configureStore } from '@reduxjs/toolkit';
```
Comment (k12kekeal) [inline front-end/src/store/store.jsx:1]: Was there a reason this got deleted?

File: front-end/webpack.config.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-const path = require('path');
+var path = require('path');
```
Comment (k12kekeal) [inline front-end/webpack.config.js]: I'm curious about this var vs. const!

File: front-end/src/store/store.jsx (hunk 1 above)
Comment (k12kekeal) [reply front-end/src/store/store.jsx:1]: nvm, I see that you re-made the file!

File: front-end/src/auth/auth0-provider-with-history.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
-import React from "react";
```
Comment (k12kekeal) [inline front-end/src/auth/auth0-provider-with-history.js:1]: Are these changes to auth and Login just reverting to the most recent versions from development?

File: front-end/webpack.config.js (hunk 2 above)
Comment (chaomonica) [reply front-end/webpack.config.js]: I don't know where that came from - haven't touched the webpack! I pulled from development and there were two path declarations so I deleted one. I can fix that.

**Review by k12kekeal — CHANGES_REQUESTED** [review]
Hey, I just made a couple of comments on things I wasn't sure about! The var vs. const thing is no big deal - mostly I just wanna confirm that the changes to /auth and /Login are correct!

Otherwise, looks good!

File: front-end/webpack.config.js (hunk 2 above)
Comment (k12kekeal) [reply front-end/webpack.config.js]: Oh, no problem! Just go with whichever is the most recent incoming change, then~

File: front-end/src/store/store.jsx (hunk 1 above)
Comment (chaomonica) [reply front-end/src/store/store.jsx:1]: I'm so confused now @.@ I still see that line on my vs code...hmm, let me try pushing again.

**Review by k12kekeal — APPROVED** [review]

**Review by k12kekeal — APPROVED** [review]

File: front-end/src/auth/auth0-provider-with-history.js (hunk 3 above)
Comment (chaomonica) [reply front-end/src/auth/auth0-provider-with-history.js:1]: I think so - just did another pull and it looks like it's up to date.

**Pull request merged**